mule-migration-tool-library/src/test/java/com/mulesoft/tools/migration/library/mule/domain/DomainTest.java

```diff
@@ -160,8 +160,14 @@ public void setUp() throws Exception {
     when(appModel.getNodes(any(String.class)))
         .thenAnswer(invocation -> getElementsFromDocuments((String) invocation.getArguments()[0]));
     when(appModel.getNode(any(String.class)))
-        .thenAnswer(invocation -> getElementsFromDocuments((String) invocation.getArguments()[0]).iterator()
-            .next());
+        .thenAnswer(invocation -> {
```

Comment:
Was this broken by your changes? It looks unrelated.

Reply:
Yes without the change i got a nullpointer